packages/sync/src/modules/Terms.php

```diff
@@ -275,4 +285,22 @@ public function expand_term_taxonomy_id( $args ) {
 			'previous_end' => $previous_end,
 		);
 	}
+
+	/**
+	 * Appends a term object to a row object from the term_relationships table.
+	 * This aids WordPress.com in re-syncing term_relationship tables that are out of sync.
+	 *
+	 * @access public
+	 *
+	 * @param object $relationship A row object from the term_relationships table.
+	 * @return object A row object from the term_relationships table with corresponding term object appended.
+	 */
+	public function add_term_to_relationship( $relationship ) {
+		return (object) array(
```

Comment:
Question: would it make sense to just mutate the existing object and add `->term` instead of creating a new array and converting it to object?

Reply:
good call, i've optimized here.